dotcom-rendering/src/web/layouts/lib/stickiness.tsx

```diff
@@ -57,7 +57,5 @@ export const SendToBack = ({ children }: Props) => (
 );
 
 export const BannerWrapper = ({ children }: Props) => (
-	<aside id="bottom-banner" css={bannerWrapper}>
-		{children}
-	</aside>
+	<aside css={bannerWrapper}>{children}</aside>
```

Comment:
Now that we're not using the id there's no need for this tag to live here so could be dropped down into the component?

Reply:
Down into the component it wraps. This tag was only put in because we needs a wrapper div to attach an id to but now that requirement is redundant

Reply:
It also has some styling applied to it. You’d move all of that out of here?